forge-gui-mobile/src/forge/adventure/scene/DeckSelectScene.java

```diff
@@ -168,6 +167,8 @@ public void enter() {
                 buttons.get(i).layout();
             }
         }
+        GameHUD.getInstance().pauseMusic();
+        SoundSystem.instance.setBackgroundMusic(MusicPlaylist.MENUS);
```

Comment:
Oh, good call, I was wondering about that! :) Thanks! I tweaked some of the instances to your solution and that seems to make the music seamless. However, it'll still reset when clicking "Back" and exiting the scene (e.g. the event or inn scene), even if the track is the same. Not sure why, but it looks like it's not caused by something I did... Or maybe I misunderstand something :)

Reply:
I think exiting the scene unloads the secondary music, it should be if exiting to worldstage and play the paused main music, and  not from the inn scene going back to town though (since the inn is inside the town)

Reply:
Yeah, not sure why it does that with the inn scene :/

Reply:
Ah, I think I found it, nvm - I'll test shortly

Reply:
Yeah, fixed :) Now the music is seamless and doesn't double itself as well. Thanks for help! :+1: